config/locales/en.yml

```diff
@@ -446,3 +446,14 @@ en:
       reduce_requests: Used to reduce the number of requests.
       save: Save cookie settings
       accept_cookies: Do you want to accept analytics cookies?
+  layouts:
```

Comment:
I think these are in layouts.yml, which needs to be renamed as en.layouts.yml to be picked up properly?

Reply:
Ooohhhh.... of course, sorry, that makes loads of sense.